src/kinetics/RxnRates.cpp

```diff
@@ -99,11 +102,15 @@ void Plog::validate(const std::string& equation)
                 // k is NaN. Increment the iterator so that the error
                 // message will correctly indicate that the problematic rate
                 // expression is at the higher of the adjacent pressures.
-                throw CanteraError("Plog::validate",
-                    "Invalid rate coefficient for reaction '{}'\nat P = {}, T = {}",
-                    equation, std::exp((++iter)->first), T[i]);
-            }
-        }
+                 // expression is at the higher of the adjacent pressures.
+                format_to(err_reactions, "\nInvalid rate coefficient for reaction '{}'\nat P = {} , T = {}",
+                    equation, std::exp((iter)->first), T[i]);
```

Comment:
Do you still need to increment the iterator here?

Reply:
Because of the change in `RxnRate.h` line 293, the iterator does not have to be incremented here